src/app/Settings/AutomatedAnalysisConfig.tsx

```diff
@@ -108,4 +108,5 @@ export const AutomatedAnalysisConfig: UserSetting = {
   description:
     'Set the recording configuration for automated analysis recordings. You may want smaller or larger values for max-age and max-size depending on how recent you want events to be recorded from the analysis.',
   content: Component,
+  category: 'Dashboard',
```

Comment:
Maybe instead of hardcording category names, there should be enum somewhere of possible categories for better typing and safeguarding?

e.g. 
`DashboardCategories.DASHBOARD = 'Dashboard'`

Also can some of these hard-coded strings start to be put into the localization files? At the least the new ones that we are introducing.

The workflow is:
1. const [t] = useTranslation();
2. (in this case): 
```js
category: t('Dashboard', {ns: 'common'}); //since this seems to be a string we will use commonly throughout Cryostat.
```
3. Then use `yarn localize` to generate the translation key in the `locales/en/common.js` files
4. Edit the json file key to whatever you wanted to hardcode.

Reply:
> Maybe instead of hardcording category names, there should be enum somewhere of possible categories for better typing and safeguarding?

Ahh for this I was using string union type so only those strings can be used. Just a thought that we are having a lot of circular dependencies so I am just avoiding importing enums.

Reply:
> Also can some of these hard-coded strings start to be put into the localization files? At the least the new ones that we are introducing.

Opps. Totally forgot to add after rebasing XD Adding now.

Reply:
Changes are becoming quite large to track so I will open a follow-up PR after some more readings and tryouts with alternatives to dynamic keys.